pkg/model/api/v1/secret/tls_config.go

```diff
@@ -84,4 +84,8 @@ type TLSConfig struct {
 	ServerName string `yaml:"serverName,omitempty" json:"serverName,omitempty"`
 	// Disable target certificate validation.
 	InsecureSkipVerify bool `yaml:"insecureSkipVerify" json:"insecureSkipVerify"`
+	// Minimum acceptable TLS version.
+	MinVersion string `yaml:"minVersion" json:"minVersion" default:"TLS12"`
```

Comment:
😲 is that default really working? I'd doubt that the config library take it into account. Specially if you use env variable to configure as lamenv library doesn't support it for sure. 
We always use the verify method to setup defaults. I think for the env var and for consistency, we should use the same here.